### PR TITLE
chore: prepare for Yggdrasil with deprecations of changing API

### PR DESCRIPTION
**Pull request by daveleek**
Deprecations in preparation of [Yggdrasil PR](https://github.com/Unleash/unleash-client-dotnet/pull/224) that removes some public APIs

### CODE REVIEW

File: src/Unleash/Utilities/ToggleBootstrapUrlProvider.cs [hunk 1]
```diff
@@ -31,6 +31,7 @@ public ToggleBootstrapUrlProvider(string path, HttpClient client, UnleashSetting
             this.customHeaders = customHeaders;
         }
 
+        [Obsolete("Will return json string in the next major version", false)]
```
Comment (sighphyre) [inline src/Unleash/Utilities/ToggleBootstrapUrlProvider.cs:34]: Think we should do a different method rather than changing the return type

**Review by sighphyre — APPROVED** [review]
Yeah LGTM

**Comment by rcollette** [comment]
What is the replacement for `.FeatureToggles`?

**Comment by sighphyre** [comment]
> What is the replacement for `.FeatureToggles`?

There likely won't be a direct replacement. To be completely honest there's still some debate to be had here. The two options (not necessarily competing) are to either return a raw JSON string, which is the unparsed response directly from the server, or a list of names of each of the feature toggles known.

IMO exposing the feature toggles directly was never a very good idea, outside of exposing their names.

I'm open to suggestions if you're using this in some way we hadn't considered yet though

**Pull request merged**